tests/python/pants_test/build_graph/test_build_graph.py

```diff
@@ -247,19 +248,18 @@ def inject_address_closure(self, spec):
     self.build_graph.inject_address_closure(Address.parse(spec))
 
   def test_invalid_address(self):
-    with self.assertRaisesRegexp(AddressLookupError, '^.* does not contain any BUILD files.$'):
+    with self.assertRaisesRegexp(AddressLookupError, '.* does not contain any BUILD files.'):
       self.inject_address_closure('//:a')
 
     self.add_to_build_file('BUILD',
                            'target(name="a", '
                            '  dependencies=["non-existent-path:b"],'
                            ')')
-    with self.assertRaisesRegexp(BuildGraph.TransitiveLookupError,
-                                 '^.*/non-existent-path does not contain any BUILD files.'
-                                 '\s+when translating spec non-existent-path:b'
-                                 '\s+referenced from //:a$'):
```

Comment:
The "referenced from" was a useful assert which shouldn't be removed (or at least should have a TODO to re-add)

Reply:
Skipped with #4515